client/map_client.go

```diff
@@ -47,11 +47,11 @@ func NewMapClientFromTree(client trillian.TrillianMapClient, config *trillian.Tr
 func (c *MapClient) GetAndVerifyLatestMapRoot(ctx context.Context) (*types.MapRootV1, error) {
 	rootResp, err := c.Conn.GetSignedMapRoot(ctx, &trillian.GetSignedMapRootRequest{MapId: c.MapID})
 	if err != nil {
-		return nil, status.Errorf(status.Code(err), "GetSignedMapRoot(%v): %v", c.MapID, err)
+		return nil, status.Errorf(status.Code(err), "GetSignedMapRoot(%v): %v", c.MapID, status.Convert(err).Message())
```

Comment:
Nice one. The `err` is parsed twice though, sadly. Probably okay, if we normally don't expect errors.

Reply:
I know... I wasn't able to find a better way.

Reply:
I found a better way:

```go
s := status.Convert(err)
return nil, status.Errorf(s.Code(), "GetSignedMapRoot(%v): %v", c.MapID, s.Message())
```